### PR TITLE
Here i change line 19 n 80 n still get error

### PR DESCRIPTION
**Pull request by arpori**
thank for u help, i use win 7, and  wamp server

### CODE REVIEW

File: lib/scaffold.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ function Scaffold($project, $table_name, $table_info) {
 
 	function list_page() {
 		$column_array = array();
-		$return_string = "<?php
+		$return_string = "<?php\n
```
Comment (tute) [inline lib/scaffold.php:18]: Hi!  `\n` should replace the "real" new line char at the end of this line, so it should read:

```
$return_string = "<?php\ninclude('../inc.functions.php');\n\n";
```

File: lib/statics/inc.config.php [hunk 2]
```diff
@@ -4,8 +4,8 @@
 /* MySQL */
 $mysql_host = 'localhost';
 $mysql_user = 'root';
-$mysql_pass = '';
-$dbname = 'phpsc_db';
+$mysql_pass = 'joko1922';
+$dbname = 'kontrak';
```
Comment (tute) [inline lib/statics/inc.config.php]: Please don't commit this. Change it again in a following commit, and you may git ignore this file (should be in the repo, I'll do it).

**Comment by tute** [comment]
Thank you @arpori! I added some comments so you can make it work and learn! :-)
You may find the error in other lines, and you'll know how to continue fixing it.
All the best!
Tute.

**Comment by arpori** [comment]
thank for help me learn PHP mr Tute..., wait for the fix, All the best!

**Comment by tute** [comment]
Closing this Pull Request, as it's not finished and I can't replicate the problem with current code.

**Pull request closed without merging**